tests/basic.test.tsx

```diff
@@ -1,6 +1,6 @@
 import { StrictMode, useEffect, useRef, useState } from 'react'
-import { expect, it, jest } from '@jest/globals'
 import { fireEvent, render, waitFor } from '@testing-library/react'
+import { vi } from 'vitest'
```

Comment:
Can we keep importing `expect` and `it` from `vitest`?
And, will it avoid adding types in tsconfig.json?

Reply:
Yes, it will avoid. I did it, but it seems to show some testing-library errors `TestingLibraryElementError: Found multiple elements with the text: button`

Reply:
Fixed

Reply:
Nice!!